game-core/src/main/java/games/strategy/triplea/ai/weak/WeakAi.java

```diff
@@ -303,7 +298,9 @@ private static void doMove(final List<MoveDescription> moves, final IMoveDelegat
           // move toward the start of the amphib route
           if (firstSeaZoneOnAmphib != null) {
             final Route r = getMaxSeaRoute(data, t, firstSeaZoneOnAmphib, player);
```

Comment:
It may be worth marking `getMaxSeaRoute` as nullable, it would allow for IDE to help us detect such NPE cases.

Reply:
Done.